xml/System.IO/File.xml

```diff
@@ -7551,10 +7679,33 @@ It is not possible to change the compression status of a <xref:System.IO.File> o
         <Parameter Name="contents" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="1" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="path">To be added.</param>
-        <param name="contents">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="path">The file to write to.</param>
+        <param name="contents">The characters to write to the file.</param>
+        <summary>
+          <para>Creates a new file, writes the specified string to the file, and then closes the file.</para>
+          <para>If the target file already exists, it is truncated and overwritten.</para>
+        </summary>
+        <remarks>
+          <para>This method uses UTF-8 encoding without a Byte-Order Mark (BOM), so using the GetPreamble method will return an empty byte array. If it is necessary to</para>
```

Comment:
```suggestion
          <para>This method uses UTF-8 encoding without a Byte-Order Mark (BOM), so using the <xref:System.Text.Encoding.GetPreamble%2A> method will return an empty byte array. If it is necessary to</para>
```

Reply:
This isn't markdown, so you would need the `<see cref...` syntax here

Reply:
Oh right, I copied it from the overload whose remarks are in markdown:
https://github.com/dotnet/dotnet-api-docs/blob/ca86f0b2a4b3d16140f7c184f0a6c03ab68c60b3/xml/System.IO/File.xml#L7775C80-L7775C122